LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
```suggestion
The MIT License (MIT)
```

Reply:
This was the only inconsistency I found with our other project licenses

Reply:
hm, OK, I copied the license from here https://github.com/zeit/fetch/blob/master/LICENSE